server/services/player.ts

```diff
@@ -0,0 +1,99 @@
+import { Request } from 'express';
+import { Match, Player, User } from '../models';
+import updatedPLayerSchema from '../validations/playerSchema';
+
+const getPlayerService = async (
+  id: number,
+): Promise<{
+  status: number;
+  data: string | object;
+}> => {
+  const player = await User.findOne({
+    where: { id },
+    attributes: ['username', 'phone', 'id'],
+    include: [
+      {
+        model: Player,
+      },
+    ],
+  });
+  if (!player) {
+    return {
+      status: 401,
+      data: 'هذا اللاعب غير موجود',
+    };
+  }
+
+  return {
+    status: 200,
+    data: player,
+  };
+};
+
+const updatePlayerService = async (
+  req: Request,
+): Promise<{
+  status: number;
+  data: string | object;
+}> => {
+  // const { UserId } = req.UserData;
+
+  const playerId = 5;
```

Comment:
Why this `id` is static it must be coming from `req.body`

Reply:
just for testing until protected routes are created